pkgs/by-name/ra/radarr/package.nix

```diff
@@ -0,0 +1,203 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  sqlite,
+  withFFmpeg ? true, # replace bundled ffprobe binary with symlink to ffmpeg package.
```

Comment:
replacing the bundled ffprobe binary is not support and breaks radarr functionality.  the same applies for sonarr

custom binaries are purpose shipped.